utils/install-dependencies.sh

```diff
@@ -46,7 +46,7 @@ function install_dependencies_with_aur() {
 function install_dependencies_with_yum() {
     sudo yum install -y yum-utils
 
-    local common_dep="curl git gcc openresty-openssl111-devel unzip pcre pcre-devel openldap-devel"
+    local common_dep="curl wget git gcc openresty-openssl111-devel unzip pcre pcre-devel openldap-devel"
```

Comment:
It seems we don't need to modify here?

The `common_dep` here is for the CI runtime environment and only needs to be installed if it is not present in the CI.

Reply:
utils/linux-install-luarocks.sh will execute wget command to get luarock tar file.
```bash
# utils/install-dependencies.sh
function install_luarocks() {
    curl https://raw.githubusercontent.com/apache/apisix/master/utils/linux-install-luarocks.sh -sL | bash -
}
```

```bash
# utils/linux-install-luarocks.sh
wget https://github.com/luarocks/luarocks/archive/v"$LUAROCKS_VER".tar.gz
```

Reply:
ok